### PR TITLE
Rename contracts to contract in template

### LINKED ISSUE #1971: Rename "contracts" to "contract" in protostar cairo1 template
In the protostar docs we can see how a single-contract project looks:
```
my_project/
├── hello_starknet/
│   ├── src/
│   │   ├── business_logic/
│   │   │   └── utils.cairo
│   │   ├── contracts/
│   │   │   └── hello_starknet.cairo
│   │   ├── business_logic.cairo
│   │   ├── contracts.cairo
│   │   └── lib.cairo
│   └── cairo_project.toml
├── tests/
│   ├── test_hello_starknet.cairo
│   └── test_utils.cairo
└── protostar.toml
```
This is a multi-contract project:
```
my_project/
├── package1/
│   ├── src/
│   │   ├── contracts/
│   │   │   └── hello_starknet.cairo
│   │  ...
│   │   ├── contracts.cairo
│   │   └── lib.cairo
│   └── cairo_project.toml
├── package2/
│   ├── src/
│   │   ├── contracts/
│   │   │   └── other_contract.cairo
│   │  ...
│   │   ├── contracts.cairo
│   │   └── lib.cairo
│   └── cairo_project.toml
...
└── protostar.toml
```

In the single-contract project template, there's a file named `contracts.cairo` and a folder named `contracts`. Such naming indicates that additional contracts should be put there while in fact they never should because this would be a compilation error and if you want to add additional contracts, you have to structure them differently.

I find it extremely misleading and I think it should be renamed "contracts" => "contract".

### PR DESCRIPTION
**Pull request by piotmag769**
### Checklist
- [x] Relevant issue is linked
- [x] Docs updated/issue for docs created
- [ ] Added relevant tests

### CODE REVIEW

File: website/.gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Dependencies
 /node_modules
+package-lock.json
```
Comment (karol-bisztyga) [inline website/.gitignore:3]: is this related?

**Review by karol-bisztyga — APPROVED** [review]

File: website/.gitignore (hunk 1 above)
Comment (piotmag769) [reply website/.gitignore:3]: Nah, but this is really annoying to me (I have to use npm instead of yarn), don't think creating another PR for just that one line is worth it
